### PR TITLE
Add error for sendRawTransaction and estimateGas

### LINKED ISSUE #3122: sendRawTransaction should return an error on invalid RLP
right now it just returns `0x0` hash.

### PR DESCRIPTION
**Pull request by tomusdrw**
Closes #3122

### CODE REVIEW

File: rpc/src/v1/helpers/errors.rs [hunk 1]
```diff
@@ -271,6 +272,14 @@ pub fn from_transaction_error(error: EthcoreError) -> Error {
 	}
 }
 
+pub fn from_rlp_error(error: DecoderError) -> Error {
```
Comment (rphmeier) [inline rpc/src/v1/helpers/errors.rs]: why are so many of the functions in the `error` module suffixed with `error`? it makes it a bit awkward when calling (`error::from_rlp_error()` vs `error::from_rlp`) as well as being inconsistent with other functions in the module, e.g. `error::signer_disabled`

Comment (gavofyork) [inline rpc/src/v1/helpers/errors.rs]: maybe make an issue to cleanup this error naming?

Comment (arkpar) [inline rpc/src/v1/helpers/errors.rs]: `error::from_rlp` looks time as error is being converted from `Rlp`

Comment (rphmeier) [inline rpc/src/v1/helpers/errors.rs]: right, it should be `error::bad_rlp` or something like that

**Pull request merged**